zookeeper-contrib/zookeeper-contrib-loggraph/src/main/java/org/apache/zookeeper/graph/servlets/JsonServlet.java

```diff
@@ -30,7 +30,7 @@
 
 import java.util.Map;
 
-abstract public class JsonServlet extends HttpServlet {
+public abstract class JsonServlet extends HttpServlet {
     abstract String handleRequest(JsonRequest request) throws Exception;
 
```

Comment:
There is no essential difference between `public abstract`  and `abstract public`， just writing style.

Reply:
I think we should follow the Java specification, Modifiers should be declared in the correct order.

> The Java Language Specification recommends listing modifiers in the following order:
Annotations
public
protected
private
abstract
static
final
transient
volatile
synchronized
native
default
strictfp